src/style.js

```diff
@@ -49,6 +53,9 @@ const style = css`
     order: -1;
     padding: 0 16px 8px 16px;
   }
+  ha-card[fill].sections .graph__legend {
```

Comment:
Why do the margins and paddings need tweaking? If it's kind of a refactor but yields the same layout, let's consider to change this also in non-section view to keep consistency.